app/assets/javascripts/components/lesson_plan.js.jsx

```diff
@@ -0,0 +1,136 @@
+//= require react-sortable-hoc/dist/umd/react-sortable-hoc.js
+
+const {SortableContainer, SortableElement, SortableHandle, arrayMove} = window.SortableHOC;
+
+const LessonPlan = createReactClass({
+  getInitialState: function() {
+    return {
+      lessons_count: this.props.lessons_count,
+      duration_in_words: this.props.duration_in_words,
+      lessons: this.props.lessons
+    };
+  },
+
+  persistLessonPlan: function(data) {
+    self = this;
```

Comment:
Sorry I didn't catch this before... Can you fix the variable scoping here in #460?

Reply:
Oops, yes. Linting would be handy for this!